erpnext_documentation/www/docs/user/manual/en/support/issue.md

```diff
@@ -32,10 +32,13 @@ replied, its status becomes "Replied".
     * Open: The Issue has been created and yet to be replied to.
     * Replied: A reply has been sent to the Issue.
     * Hold: The Issue is on Hold due to some reason.
+    * Resolved: When the users are reasonably sure that they have provided the customer with a solution to their problem but have not acknowledgement about the resolution from the customer.
     * Closed: The Customer got a resolution and the Issue was closed.
 
     If the sender replies to the thread, the status becomes "Open" again. User can "Close" the Issue manually by clicking on the **Close** button on the top right.
 
+> Note: If SLA has been set up, then the fulfillment status of the SLA will be updated on both, the **Closed** as well as the **Resolved** status. The key difference between these two statuses is to indicate the case where the user solving the issue is sure about the resolution and the customer has not yet acknowledged that the issue has been resolved (Resolved) and the case where an acknowledgement is received (Closed).
```

Comment:
If SLA has been set up, then the fulfillment status of the SLA will be updated on both **Closed** as well as **Resolved** status. 

The second line can be removed as you are already adding that above.